examples/showcase/anatomy.py

```diff
@@ -0,0 +1,143 @@
+# This figure shows the name of several matplotlib elements composing a figure
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Circle
+from matplotlib.ticker import MultipleLocator, FuncFormatter
+
+np.random.seed(123)
+
+X = np.linspace(0.5, 3.5, 100)
+Y1 = 3+np.cos(X)
+Y2 = 1+np.cos(1+X/0.75)/2
+Y3 = np.random.uniform(Y1, Y2, len(X))
+
+plt.figure(figsize=(8, 8), facecolor="w")
```

Comment:
I think both ways are fine, But in the current I would replace the "matlab like" shorthand syntax 
```
ax = plt.subplot(111, aspect=1)
```
with the more explicit version
```
ax = plt.subplot(1, 1, 1, aspect=1)
```
http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.subplot